hummingbot/strategy/pure_market_making/constant_multiple_spread_pricing_delegate.pyx

```diff
@@ -22,18 +22,34 @@ cdef class ConstantMultipleSpreadPricingDelegate(OrderPricingDelegate):
     def bid_spread(self) -> Decimal:
         return self._bid_spread
 
+    @bid_spread.setter
+    def bid_spread(self, value):
+        self._bid_spread = value
+
     @property
     def ask_spread(self) -> Decimal:
         return self._ask_spread
 
+    @ask_spread.setter
+    def ask_spread(self, value):
+        self._ask_spread = value
+
     @property
     def number_of_orders(self) -> int:
         return self._number_of_orders
 
+    @number_of_orders.setter
```

Comment:
Do we still need setters for `number_of_orders` and `order_interval_size` if only bid and ask spreads are modifiable?

Reply:
no, not needed, I'll remove them.